src/models/ministryPlate.ts

```diff
@@ -79,7 +81,7 @@ export class MinistryPlateDocument extends DocumentModel {
     const plateData: Partial<MinistryPlate> = {
       plateSerialNumber: plate.plateSerialNumber,
       dtpNumber: techRecord.techRecord_brakes_dtpNumber,
-      primaryVrm: techRecord.primaryVrm ?? techRecord.trailerId,
+      primaryVrm: techRecord.techRecord_vehicleType === 'hgv' ? techRecord.primaryVrm : techRecord.trailerId,
```

Comment:
```suggestion
      primaryVrm: techRecord.techRecord_vehicleType === 'trl' ? techRecord.trailerId : techRecord.primaryVrm,
```

Reply:
Not touching plates more that I need too, sorry.